### PR TITLE
APP-5312 - serialize data collection methods in upper camel case

### PR DESCRIPTION
**Pull request by gvaradarajan**

### CODE REVIEW

**Review by mattjperez — APPROVED** [review]
LGTM

**Comment by maxhorowitz** [comment]
only concern is that this change should be made for all built-in sensor data capture method types - are those 4 the only that exist?

**Comment by gvaradarajan** [comment]
> only concern is that this change should be made for all built-in sensor data capture method types - are those 4 the only that exist?

yes, for now. I'll add a comment so that future methods are done the right way

File: micro-rdk/src/common/data_collector.rs [hunk 1]
```diff
@@ -69,12 +69,14 @@ pub enum CollectionMethod {
 
 impl Display for CollectionMethod {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        // NOTE: Future added methods need to follow the same upper camel case
```
Comment (maxhorowitz) [inline micro-rdk/src/common/data_collector.rs:72]: Thank you!

**Review by maxhorowitz — COMMENTED** [review]
:shipit:

**Review by maxhorowitz — APPROVED** [review]
:shipit:

**Pull request merged**